src/Raven.Server/Web/HtmlUtil.cs

```diff
@@ -65,8 +65,23 @@ private static string GetUnsecuredAccessFlagsHtml()
             return unsecuredAccessFlagsHtml;
         }
         
-        public static string RenderStudioAuthErrorPage(string reason, string reasonAddon)
+        public static string RenderStudioAuthErrorPage(string error)
         {
+            string reason;
+            string reasonAddon;
+            
+            var stringBreak = error.IndexOf(Environment.NewLine);
```

Comment:
Do we need to break this at all? Can't we simply display this and single text? Such solutions are hard to maintain in future. Btw. you can split text using .Split method on string.

Reply:
1. The break was added because it seemed more nice & clear like this.
   See  image:  https://issues.hibernatingrhinos.com/issue/RavenDB-13548#focus=Comments-67-50351.0-0
2. I didn't want to use split because I only wanted to get 2 parts for the html,
   and not an array of strings in case someone added more than one 'break' in the message text.

Reply:
split has paramater to limit items count

Reply:
awesome - done